common/app/views/fragments/commercial/paidForMeta.scala.html

```diff
@@ -1,12 +1,12 @@
-@()(implicit request: RequestHeader)
+@(dataId: String = s"")(implicit request: RequestHeader)
```

Comment:
Don't need the `s` prefix if it's a string literal.  
But if this doesn't need to be specified by the caller it would be better to have the type `Option[String]` with a default value of `None`.